src/Register.php

```diff
@@ -67,8 +67,9 @@ public static function register_widgets() {
 	private static function register_widget( $widget_class ) {
 		register_widget( $widget_class );
 
-		if ( method_exists( $widget_class, 'post_registration' ) ) {
-			call_user_func( [ $widget_class, 'post_registration' ] );
+		$callback = [ $widget_class, 'post_registration' ];
+		if ( is_callable( $callback, true ) ) {
```

Comment:
It failed in wp-elements. There the function is static, which is probably the reason.